### PR TITLE
Install all the things, slap some styles onto that App

### PR DESCRIPTION
**Pull request by marwinburesch**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import styled from 'styled-components';
```
Comment (macfrei) [inline src/App.js]: Could we import from `styled-components/macro` please? :)

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { createGlobalStyle } from 'styled-components';
```
Comment (macfrei) [inline src/GlobalStyles.js]: Here as well

File: src/Header.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+export default function Header() {
+    return (
+        <>
+            <Container class="page-header round-top shadow">
```
Comment (macfrei) [inline src/Header.js]: Do we need those class names?

File: src/Header.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
```
Comment (macfrei) [inline src/Header.js]: Here as well :)

File: src/Header.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+export default function Header() {
+    return (
+        <>
```
Comment (macfrei) [inline src/Header.js]: I don't think we'll need those fragments here as the elements are within the `Container`

File: src/Page.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
```
Comment (macfrei) [inline src/Page.js]: Here as well :D

**Review by macfrei — CHANGES_REQUESTED** [review]
I added some comments, mostly small things. It'd be great if you could import from `styled-components/macro` instead of `styled-components` :)

**Comment by marwinburesch** [comment]
> I added some comments, mostly small things. It'd be great if you could import from `styled-components/macro` instead of `styled-components` :)

Fixed
Thank you very much!

File: src/Header.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                >
+                    <path
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        stroke-width="2"
+                        d="M4 6h16M4 12h16M4 18h16"
+                    />
+                </svg>
```
Comment (marwinburesch) [inline src/Header.js:21]: I am considering to just import this, also I just saw a `class` 😱

Comment (macfrei) [reply src/Header.js:21]: :O :O :O

Comment (marwinburesch) [reply src/Header.js:21]: Imma fix this later

**Pull request merged**